R/modules.R

```diff
@@ -416,10 +414,10 @@ format.teal_module <- function(
       content_prefix, "|- ", crayon::green("Server Arguments : "), server_args_formatted, "\n"
     )
   }
-  if ("transformers" %in% what) {
+  if ("transformators" %in% what) {
     output <- paste0(
       output,
-      content_prefix, "L- ", crayon::magenta("Transformers     : "), transformers, "\n"
+      content_prefix, "L- ", crayon::magenta("transformators       : "), transformators, "\n"
```

Comment:
did you know that `crayon` is superseded package? [link](https://cran.r-project.org/web/packages/crayon/index.html#:~:text=package%20is%20now-,superseded,-.%20Please%20use%20the)

Maybe it's the right time to remove it?

Reply:
@pawelru we barely introduced it : D https://github.com/insightsengineering/teal/pull/1401
CC @vedhav

Reply:
lol :)
let's find an alternative

Reply:
ok, created an issue for this in here https://github.com/insightsengineering/teal/issues/1417